app/src/main/shared/components/ReduxForm/MontaguForm.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
```

Comment:
What do you think of this HOC? Have applied it to the create user and create modelling groups forms as well as the new touchstone form. Do you think it makes the forms simpler to write or is it unnecessary ?